packages/matter-node.js/tsconfig.json

```diff
@@ -12,6 +12,7 @@
         "noUnusedParameters": true,
         "noUnusedLocals": true,
         "strict": true,
+        "resolveJsonModule": true,
```

Comment:
Why you added this? Which json files are resolved?

Reply:
Didn't mean to leave that in.  I needed it for the following on newapi branch...  Not sure why it worked for you but not me, maybe node version (I'm on 20)?  Regardless, I can pull it out if you want.

```typescript
import packageJson from "../package.json"
```

Reply:
Yes I will kill those again ... no idea why they came in again :-)